setup.cfg

```diff
@@ -98,6 +98,8 @@ invenio_assets.webpack =
     invenio_vocabularies = invenio_vocabularies.webpack:theme
 invenio_i18n.translations =
     invenio_vocabularies = invenio_vocabularies
+invenio_celery.tasks =
+    invenio_vocabularies_services = invenio_vocabularies.services.tasks
```

Comment:
This change was not strictly necessary due to the import statement `from .tasks import process_datastream` in `invenio_vocabularies/services/service.py`.
However, we should define it here to make sure it always stays discoverable.

Please note that the other existing task was in a `services` directory and I added the task to the same place.
Let me know if you'd prefer to have the task defined one level higher to be more standard.